hangman.py

```diff
@@ -137,9 +204,18 @@ def match_with_gaps(my_word, other_word):
         _ , and my_word and other_word are of the same length;
         False otherwise: 
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
-
+    my_word_new = my_word.replace(' ', '')
+    if len(my_word_new) == len(other_word):
+        for i in range(len(my_word_new)):
+            if my_word_new[i] == other_word[i]:
+                continue
+            elif my_word_new[i] == "_" and other_word[i] not in list(my_word_new):
```

Comment:
You use that "_" symbol not only once. And it has some meaning for the game. Maybe it makes sense to have a constant for that?